### PR TITLE
Add Clickhouse vulnerabilities

### PR DESCRIPTION
**Pull request by srmish-jfrog**

### CODE REVIEW

File: 2021/37xxx/CVE-2021-37138.json [hunk 1]
```diff
@@ -1,18 +1,78 @@
 {
```
Comment (cve-team) [inline 2021/37xxx/CVE-2021-37138.json]: CNA(s) of user (JFROG) do not match owning CNA.

File: 2021/37xxx/CVE-2021-37139.json [hunk 2]
```diff
@@ -1,18 +1,78 @@
 {
```
Comment (cve-team) [inline 2021/37xxx/CVE-2021-37139.json]: CNA(s) of user (JFROG) do not match owning CNA.

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/37xxx/CVE-2021-37138.json (hunk 1 above)
Comment (srmish-jfrog) [reply 2021/37xxx/CVE-2021-37138.json]: This was a legacy CVE ID, changed to CVE-2021-43304

File: 2021/37xxx/CVE-2021-37139.json (hunk 2 above)
Comment (srmish-jfrog) [reply 2021/37xxx/CVE-2021-37139.json]: This was a legacy CVE ID, changed to CVE-2021-43305

**Pull request merged**